charts/ui-plugin-server/Chart.yaml

```diff
@@ -1,18 +1,21 @@
 annotations:
   catalog.cattle.io/certified: rancher # Any application we are adding as a helm chart
+  catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.25.0-0'
   catalog.cattle.io/namespace: cattle-ui-plugin-system # Must prefix with cattle- and suffix with -system=
-  catalog.cattle.io/ui-component: plugins # This is added for custom UI deployment of a chart
   catalog.cattle.io/os: linux
   catalog.cattle.io/permits-os: linux, windows
-  # catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'
-  # catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.25.0-0'
+  catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'
   catalog.cattle.io/scope: management
+  catalog.cattle.io/ui-component: plugins
 apiVersion: v1
-appVersion: "1.16.0"
-description: A base Plugin Server Chart for adding plugins to Rancher
-name: test-plugin
+appVersion: "0.1.0"
```

Comment:
these are automatically written from the package.json, see the new functionality of the patch script